tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -29,6 +29,9 @@ ulimit -a
 pip install --user google-api-python-client oauth2client
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/GrpcTesting-d0eeee2db331.json
 
+# Upgrade pip to understand manylinux2010 wheels
+pip install -U pip
```

Comment:
Perhaps pin `pip`/`virtualenv` so the build environment doesn't accidentally change underneath you?

Reply:
Good catch.

The installed pip is `19.0.?` in Kokoro. This line of upgrading doesn't have the expected value. I maybe remove it in latter commits.